cli/injector/cpu_pressure.go

```diff
@@ -18,10 +21,40 @@ var cpuPressureCmd = &cobra.Command{
 	PreRun: func(cmd *cobra.Command, args []string) {
 		// prepare spec
 		spec := v1beta1.CPUPressureSpec{}
+		podCores, err := getPodCores()
+		if err != nil {
+			log.Fatalw("error creating the node injector", "error", err)
+			return
```

Comment:
```suggestion
			return
```

Fatal imply `os.Exit(1)`, return is not necessary

Reply:
I would also add that we should not fatal in the `PreRun`. The injector is made so it does not exit (especially something else than exit 0) on error during the injection phase because otherwise it would prevent the disruption to be deleted properly while there's no need to block the removal if injection couldn't be done.

It is why most of this logic is supposed to be done in the injector (and not in the cli itself) so it's handled later as a "normal error" and not a fatal thing.

Reply:
I will think a bit more about error handling.